### PR TITLE
Improve Romeo Asset Token Contract 

### PR DESCRIPTION
**Pull request by setzeus**
## Description
Tiny PR meant to address issues opened in #89 with an additional fix (unnecessary optional). 

## Checklist
- [x] Updated sBTC supply limit to correct amount
- [x] Updated JSON asset link URI
- [x] Removed unnecessary optional

### CODE REVIEW

**Comment by will-at-stacks** [comment]
Drop the 'r'.  It's cleaner.  (Romeo)

File: romeo/asset-token.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "name":	"sBTC Romeo Token",
```
Comment (friedger) [inline romeo/asset-token.json:2]: Add these properties to adhere to sip 16:
```
"sip": 16,
"decimals": 8,
"symbol": "sBTC"
```

**Review by friedger — CHANGES_REQUESTED** [review]
I'd like to see more properties for SIP-16 compatibility.

File: romeo/asset-token.json (hunk 1 above)
Comment (setzeus) [reply romeo/asset-token.json:2]: Copy, added the three (3) sip 16 properties^

**Comment by friedger** [comment]
Clarity tests fail

**Comment by jo-tm** [comment]
#112 is blocking https://github.com/stacks-network/sbtc/pull/120

**Comment by setzeus** [comment]
@friedger test fixed

@jo-tm adding you as a reviewer too to unblock faster

**Review by jo-tm — APPROVED** [review]
LGTM!

**Comment by jo-tm** [comment]
clarinet tests are good, the PR Rust Linter is breaking.

**Review by FriendsFerdinand — APPROVED** [review]

**Pull request merged**